.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Bug report
+about: Report a problem with this library that needs to be fixed.
+title: ''
+labels: triage
+assignees: ''
+
+---
+
+**Before filing this issue**
+Make sure you've carefully reviewed the
+[documentation](https://developers.google.com/google-ads/api/docs/client-libs/python)
+for this library.
+
+This library is only used for the
+[Google Ads API](https://developers.google.com/google-ads/api/docs/start), for
+the [AdWords API](https://developers.google.com/adwords/api/docs/guides/start)
+client library report your issue
+[here](https://github.com/googleads/googleads-python-lib/issues).
+
+Only use this page to report problems with this library. If you are unsure
+whether your issue is related to this library or the Google Ads API itself,
+please first reach out to one of the below resources before filing an issue here:
+
+* [Post in the Support Forum](https://support.google.com/google-ads/thread/new)
+* [Email the Support Team](https://support.google.com/google-ads/gethelp)
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Specific steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Library version and API version (please complete the following information):**
+ - API [e.g. v6]
+ - Dependencies (i.e. `pip freeze`)
+
+**Request/Response Logs**
+See [here](https://developers.google.com/google-ads/api/docs/client-libs/python/logging)
+for details on how to enable logging.
+
+**NOTE** please redact personally identifiable information from the logs,
```

Comment:
nit: capital Please

Reply:
I removed because I remember it's not good style to use "please" in documentation, at least based on the warnings I see in Critique.